motmetrics/apps/evaluateTracking.py

```diff
@@ -8,6 +8,7 @@
 from __future__ import absolute_import
 
 import argparse
+import io
```

Comment:
you may just `from io import open` so it is still simple...

Reply:
Thanks for the suggestion! I actually prefer the explicit way so that it's obvious that it's not the builtin open()